azure-governance-cloudwise/OMSAzureDashboards/OMSSQLDBAzureMonitoringSolution.omsview

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
if this is a template file, should have a .json extension.
follow the BPs for template placement (nested or a different sample)?